src/client/styles.css

```diff
@@ -0,0 +1,8 @@
+table {
```

Comment:
Should probably change this to a class in case we want to style multiple tables differently.